app/src/main/java/io/islnd/android/islnd/app/adapters/PostAdapter.java

```diff
@@ -62,9 +62,11 @@ public void onBindViewHolder(GlancePostViewHolder holder, Cursor cursor) {
         holder.postContent.setText(post.getContent());
         //TODO: Get actual comment count
         holder.postCommentCount.setText(Util.numberOfCommentsString(0));
-
-        // Go to profile on picture click
+        
```

Comment:
This assumes that the context passed in is the ProfileActivity context and not the Application context.